scimath/units/unit_db.py

```diff
@@ -201,6 +203,8 @@ def get_family_ranges_from_file(self, filename=None):
         #  manager).
 
         is_data = False
+        column_names = []
+        system_names = []
```

Comment:
Same question here: can we drop the initialization of `column_names` and `system_names` in the `else` block?

Reply:
I left them to ensure that they do get (re?)set in that `else:` block. That `else:` block _ought_ to be executed before any other branch that needs those values, and only once, but we're doing I/O, so it's never certain.

Reply:
I think it is certain in this case: the `elif` block can't run before the `else` does (because `is_data` doesn't become true until the `else` block runs), and the top `if` doesn't care.